src/Database/PostgreSQL/PQTypes/Checks.hs

```diff
@@ -81,7 +81,8 @@ migrateDatabase options
     resultCheck =<< checkExistenceOfVersionsForTables (tableVersions : tables)
 
   -- After migrations are done make sure the table versions are correct.
-  resultCheck . checkVersions options =<< getTableVersions (tableVersions : tables)
+  resultCheck . checkVersions options
+    =<< getTableVersions AllTables (tableVersions : tables)
```

Comment:
I was wondering - wouldn't it make sense to remove table versions for tables which don't exist anymore?